leash.go

```diff
@@ -516,3 +519,72 @@ func logStats(stats *responseStats, interval uint) {
 		stats.logAndReset()
 	}
 }
+
+// Final helper for outputting helpful log lines at the end of honeytail
+// execution. Calls private `responseStats` members indiscriminately
+// because we know this is only called at the end, after all goroutines
+// have exited.
+func outputHelp(options GlobalOptions, stats *responseStats, teamSlug string) {
+	// Build up Honeycomb URL for nicer logs
+	u, _ := url.Parse("https://ui.honeycomb.io")
+	u.Path = path.Join(u.Path, teamSlug, "datasets", options.Reqs.Dataset)
+
+	successes := 0
+	for code, count := range stats.totalStatusCodes {
+		if code >= 200 && code < 300 {
+			successes += count
+		}
+	}
+
+	if !options.Backfill && stats.finalAvgLagSeconds > 60*60 {
```

Comment:
Can you define `60*60` as a `const` so there's a little semantic detail there?

Why have `finalAvgLagSeconds` as `float64` instead of `time.Duration`?

I don't think the honeyelb issue we had before would have gotten picked up that way (at least in the logs I saw), the biggest delta was about 50 minutes or so. How about 30 minutes?

Reply:
> I don't think the honeyelb issue we had before would have gotten picked up that way (at least in the logs I saw), the biggest delta was about 50 minutes or so. How about 30 minutes?
**Update on this:** This is actually trickier to tackle than I had thought since when, say, `honeyelb` starts up, it ingests logs from quite a while ago _first_, and it'd be unsettling to see a bunch of yellow `WARN`s when the app is actually in normal operation and simply trying to catch up.

Might be worth checking something like "honeytail has been running for more than 2 hours" in addition to the delta being high, since at the start of ingest having an otherwise wide delta might be considered normal. Curious if you have other ideas as well.

Reply:
> Might be worth checking something like "honeytail has been running for more than 2 hours" in addition to the delta being high

Ooh, good call. FWIW the `Warn` only happens on exit (`outputHelp` is called once at the end of `run`), but +1 considering the uptime/run time or ... maybe a moving average is the right answer? Hrmm. Will think about this and push an update.